Maths/PrimaryNumber.py

```diff
@@ -0,0 +1,23 @@
+import sys
+def main():
+    print("Calculate primary number from 0 to your number:")
+    while True:
+        num = eval(input("To what number?"))
+        print('resualt:' , pr(num))
+
+def pr(num):
+    #we know 2 is a primary number then next number if undivisibie with 2, it is primary
+    #Any number that not divisible by numbers smaller than itself is primary
+    p = [2]
```

Comment:
Umm, `Primary number` sounds a little confusing, since we are more familiar with `prime number`.

And we should use more friendly names like `pr(num)` -> `generatePrimes`
`p` -> `primes`
`I` -> `number`